content/300-guides/050-database/100-developing-with-prisma-migrate/170-customizing-migrations.mdx

```diff
@@ -30,6 +23,13 @@ To edit a migration file before applying it:
    npx prisma migrate dev
    ```
 
+<Admonition type="info">
+
+This guide **does not apply for MongoDB**.<br />
+Instead of `migrate dev`, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
+
+</Admonition>
```

Comment:
Point out that Migrate for MongoDB is db push instead of "not supported" warning.